phoenix-core/src/main/java/org/apache/phoenix/query/KeyRange.java

```diff
@@ -519,7 +520,8 @@ public KeyRange union(KeyRange other) {
         return Lists.transform(keys, POINT);
     }
 
-    private static int compareUpperRange(KeyRange rowKeyRange1,KeyRange rowKeyRange2) {
+    @VisibleForTesting
```

Comment:
I'll remove @VisibleForTesting. This needs to be public, because it is used by changes that I'm working on.